styles/blocks/_gallery.scss

```diff
@@ -1,255 +1,24 @@
-$fotorama-duration-time: 0.3s;
```

Comment:
Would be great to move gallery styles to some module dir... But there is nothing like `MagentoGallery`, so probably best option will be `MagentoCatalog`.